serverless/knative_eventing/serverless-understanding-eventing.adoc

```diff
@@ -1,36 +1,42 @@
 include::modules/serverless-document-attributes.adoc[]
-[id="serverless-event-architecture"]
-= Knative Eventing architecture
-:context: serverless-event-architecture
+[id="serverless-understanding-eventing"]
+= Understanding Knative Eventing
 include::modules/common-attributes.adoc[]
+:context: serverless-understanding-eventing
 
 toc::[]
 
 Knative Eventing on {product-title} enables developers to use an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] with serverless applications.
-An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  _sinks_, or consumers, that receive them.
 
-Knative Eventing uses standard HTTP POST requests to send and receive events between event producers and consumers.
-These events conform to the link:https://cloudevents.io[CloudEvents specifications], which enables creating, parsing, sending, and receiving events in any programming language.
+[id="serverless-understanding-eventing-event-driven-arch"]
+== Event-driven architecture
 
-You can propagate an event from an xref:../event_sources/knative-event-sources.adoc#knative-event-sources[event source] to multiple event sinks by using:
+An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sinks_], or consumers, that receive them.
 
-* xref:../../serverless/channels/serverless-channels.adoc#serverless-channels[channels] and subscriptions, or
-* xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[brokers] and triggers.
+Knative Eventing uses standard HTTP POST requests to send and receive events between event producers and sinks. These events conform to the link:https://cloudevents.io[CloudEvents specifications], which enables creating, parsing, sending, and receiving events in any programming language.
```

Comment:
s/[CloudEvents specifications], which enables/[CloudEvents specifications], which enable

_Secifications_ is considered plural, right?

Reply:
I think "which enables" is right here, because we're saying "the fact that they conform to this standard enables x" as opposed to the "specifications enable x"